app/femr/ui/views/history/indexPatient.scala.html

```diff
@@ -61,21 +68,20 @@ <h4 class="text-center">Duplicate Patient Search Results</h4>
             </div>
             <div id="patientInformation">
                 <h1 class="bold margin-top-zero">Patient Id: @viewModel.getPatientItem.getId</h1>
-                @for(roles <- currentUser.getRoles()) {
 
-                    @if(roles.getId().equals(1) || roles.getId().equals(6) ) {
-                        @helper.form(action = TriageController.deletePatientPost(viewModel.getPatientItem.getId)){
-                            <input type="hidden" name="reasonDeleted" id="reasonDeleted" />
-                            <button hidden="true" type="submit"  id="deletePatient"></button>
-                        }
-                        <span>
-                            <button type="submit" id="deletePatientBtn" class="btn btn-danger pull-right"> Delete this Patient</button>
-                        </span>
+                @if(roles.contains(Roles.ADMINISTRATOR) || roles.contains(Roles.SUPERUSER) ) {
```

Comment:
i like it. at first i was hesitant because we're exposing the data layer directly to the UI, but this is how roles are managed in the controllers as well :+1: